README.md

```diff
@@ -1,2 +1,118 @@
-# strigo-github-exercises-examples
-Examples for markdown exercises that may be used in Strigo training events
+# Strigo GitHub Class Exercises examples
+
+## Quick start
+
+1. Create `.strigo/config.yml` file in your exercises' repository
+1. Configure exercises in `.strigo/config`
+
+   ```yaml
+   ---
+   classes:
+     - exercises:
+         # `file` is relative to .strigo directory
+         - file: "../Exercise.md"
+           title: "Title!"
+   ```
+
+1. In Strigo Class editor, fetch Exercises for a desired Class
+1. Done!
+
+---
+
+## Config file structure
+
+- `classes` - array of Classes
+  - `class_id` - defines the Class ID, optional if the configuration has only one Class
```

Comment:
I'd do: "The ID of the class you'd like to fetch exercises for (optional if the repo only supports one class)."

Reply:
"supports" sounds wrong to me. "contains"?

Reply:
I think "supports" actually makes more sense than contains. The repo doesn't "contain" a class.